app/javascript/courses/review/components/CoursesReview__Editor.res

```diff
@@ -1069,6 +1132,19 @@ let make = (
     pending ? GradesEditor : ReviewedSubmissionEditor(OverlaySubmission.grades(overlaySubmission))
   }
 
+  React.useEffect0(() => {
+    switch submissionReport {
+    | Some(report) => {
+        let intervalId = Js.Global.setInterval(
+          () => reloadSubmissionReport(report, updateSubmissionReportCB),
+          30000,
```

Comment:
This should be configurable via an environment variable (see related comment in specs).